content/docs/contributing/release-process.md

```diff
@@ -141,15 +143,158 @@ page if a step is missing or if it is outdated.
+     previous release note pages.
+   5. Replace the GitHub issue numbers and GitHub handles (e.g., `#1234` or
+       `@maelvls`) with actual links using the following command:
+
+       ```bash
+       sed github-release-description.md \
+         -e 's$#([0-9]+)$[#\1](https://github.com/cert-manager/cert-manager/pull/\1)$g' \
+         -e 's$@(\w+)$[@\1](https://github.com/\1)$g' \
+         -E \
+         -i
+       ```
+
+   6. Move `release-notes.md` to the website repo:
+
+      ```bash
+      # From the cert-manager repo.
+      mv release-notes.md ../website/content/docs/release-notes-1.X.md
+      ```
+
+   7. Add an entry to `content/docs/manifest.json`:
+
+        ```diff
+         {
+           "title": "Release Notes",
+           "routes": [
+        +    {
+        +      "title": "v1.12",
+        +      "path": "/docs/release-notes/release-notes-1.12.md"
+        +    },
+        ```
+
+   8. Add a line to the file `content/docs/release-notes/README.md`.
+
+5. **(final + patch release)** Prepare the Website "Release Notes" PR.
+
+   > ⚠️ This step can be done ahead of time.
+
+   In that PR:
+
+   1. (**final release**) Update the section "Supported releases" in the
+     [supported-releases](../installation/supported-releases.md) page.
+   2. (**final release**) Update the section "How we determine supported
+     Kubernetes versions" on the
+     [supported-releases](../installation/supported-releases.md) page.
+   3. (**final release**) Bump the version that appears in
+     `scripts/gendocs/generate-new-import-path-docs`. For example:
+
+      ```diff
+      -LATEST_VERSION="v1.11-docs"
+      +LATEST_VERSION="v1.12-docs"
+
+      -genversionwithcli "release-1.11" "$LATEST_VERSION"
+      +genversionwithcli "release-1.12" "$LATEST_VERSION"
+      ```
+
+   4. (**final + patch release**) Bump all versions present in installation
+     instructions. To find these versions:
+
+      ```bash
+      find ./content/docs/installation -name '*.md' -not -path '*/upgrad**' -exec sed -i.bak 's/1.11../1.12.0/g' '{}' \;
+      rm -f **/*.bak
+      ```
+
+      To check that all mentions of that versions are gone, run:
+
+      ```bash
+      grep -R -n -F 'v1.11.' content/docs/installation
+      ```
+
+   5. (**final release only**) Freeze the `docs/` folder by creating a copy ,
+     removing the pages from that copy that don't make sense to be versioned,
+     and updating the `manifest.json` file:
+
+      ```bash
+      cp -r content/docs content/v1.12-docs
+      rm -rf content/v1.12-docs/{installation/supported-releases,installation/upgrading,release-notes}
+      sed -i.bak 's|docs|v1.12-docs|g' content/v1.12-docs/manifest.json
+      jq 'del(.. | select(.path? | select(.) | test(".*(installation/supported-releases.md|installation/upgrading|release-notes).*")))' \
+         content/v1.12-docs/manifest.json >tmp && mv tmp content/v1.12-docs/manifest.json
```

Comment:
note: this will need updating to reflect the review comments in #1252

Reply:
Done! I added `jq 'del(.. | select(.routes? == []))'` to remove the empty `routes`.